proposals/11551-kubernetes-native-api/TestPlan.md

```diff
@@ -0,0 +1,87 @@
+## Feature Summary
+Please refer [this](https://github.com/kubeflow/pipelines/blob/master/proposals/11551-kubernetes-native-api/README.md) for more details
+
+## Test Sections
+### Migration of existing pipelines
+#### With the migration script
+Create few pipelines in DB mode and migrate it via the migration script
+
+| **Test Case**                                                                                                                        | **Test Steps**                                                                                                                        | **Expected Result**                                                                                          |
+|--------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------|
+| (KFP DB mode) Create a pipeline and a pipeline version and run migrate script                                                        |                                                                                                                                       | Pipeline and pipeline version should match exactly except for ID?                                            |
```

Comment:
Yes, but the ID should match in the `pipelines.kubeflow.org/original-id` annotation. Additionally, we need to verify that the `spec.pipelineName` field matches to the pipeline name on the pipeline version.